lambdas/ecs_ec2_instance_tagger/ecs_ec2_instance_tagger.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+# -*- encoding: utf-8 -*-
+"""
+On an ECS Container Instance State Change tag the EC2 instance specified by
+"ec2InstanceId" with "clusterArn" and "containerInstanceArn".
+
+This is in order that we can easily identify which cluster and
+containerInstance an EC2 instance belongs to.
+
+This script runs when triggered from an ECS Container Instance State Change.
+
+Requires a Cloudwatch Event with pattern:
+
+{
+  "source": [
+    "aws.ecs"
+  ],
+  "detail-type": [
+    "ECS Container Instance State Change"
+  ]
+}
+
```

Comment:
👍 for the nice description, especially the event pattern.

Some of the AWS terminology makes this very wordy and hard to follow though (not your fault!). Could we break it up a little? Maybe:

> Tag EC2 instances with their cluster ARN and container instance ARN. This allows us to easily follow an instance back to its ECS cluster.
>
> This script is triggered by an ECS Container Instance State Change event.